src/app/cpe/widgets/search-widget/search-widget.component.html

```diff
@@ -0,0 +1,25 @@
+<div>Search widget</div>
+<br>
+<cpe-button
+    [id]="'button-01'"
```

Comment:
from what I can tell, `id` is not an input in the `cpe-button` component so the id shouldn't be wrapped in square brackets

Reply:
Done in https://github.com/dasch-swiss/dsp-cpe/pull/11/commits/ec92e796e8dcb7b4f872614f50db5574523e988b